pkg/helpers/generate_azure_constants.py

```diff
@@ -75,7 +76,27 @@ def get_storage_account_type(size_name):
     return "Standard_LRS"
 
 
-def get_file_contents(dcos_master_map, kubernetes_size_map, locations):
+def get_accelerated_skus():
+    """
+    Returns a dict of SKU name to AcceleratedNetworkingEnabled value.
+    """
+    pinned = [
+        ("AZAP_Performance_ComputeV17C", True),
+        ("SQLGL", True),
+        ("SQLGLCore", True),
+    ]
+    query = r"[? starts_with(name, `Standard`) && !ends_with(name, `Promo`)].{name: name, caps: capabilities[? name==`AcceleratedNetworkingEnabled`]}[].{sku: name, acceleratedNetworking: caps[0].value}[? acceleratedNetworking != null]"
```

Comment:
Are we excluding promo SKUs to keep the list leaner?

Reply:
Yes, basically. The `AcceleratedNetworkingSupported` func also starts with this line before interrogating the data structure:

```go
sku = strings.TrimSuffix(sku, "_Promo")
```